app/proxy.go

```diff
@@ -75,10 +75,10 @@ func proxy(ctx context.Context, remote net.Conn, local net.Conn, config *tls.Con
 		if err != nil {
 			errs[0] = fmt.Errorf("local -> remote: %v", err)
 		}
-		tcp.CloseRead()
```

Comment:
I'd suggest to change this branch and KEEP this line that is being deleted, as explained it should avoid logging spurious errors when the dqlite engine is being shutdown and client connections get legitimately closed.

The `remote.Close()` call that this branch adds just below should be fine and will serve as extra safety net.

Reply:
Done!